addons/website/models/website_snippet_filter.py

```diff
@@ -70,20 +73,27 @@ def _prepare_values(self, limit=None, search_domain=None):
                 domain = expression.AND([domain, [('is_published', '=', True)]])
             if search_domain:
                 domain = expression.AND([domain, search_domain])
-
-            records = self.env[filter_sudo.model_id].search(
-                domain,
-                order=','.join(literal_eval(filter_sudo.sort)) or None,
-                limit=limit
-            )
-            return self._filter_records_to_dict_values(records)
+            try:
+                records = self.env[filter_sudo.model_id].search(
+                    domain,
+                    order=','.join(literal_eval(filter_sudo.sort)) or None,
+                    limit=limit
+                )
+                return self._filter_records_to_dict_values(records)
+            except MissingError:
+                _logger.warning("Unexpected domain [%s] in ir.filter, interpreted as False", expression(domain, self).result)
+                return []
         elif self.action_server_id:
-            return self.action_server_id.with_context(
-                dynamic_filter=self,
-                limit=limit,
-                search_domain=search_domain,
-                get_rendering_data_structure=self._get_rendering_data_structure,
-            ).sudo().run()
+            try:
+                return self.action_server_id.with_context(
+                    dynamic_filter=self,
+                    limit=limit,
+                    search_domain=search_domain,
+                    get_rendering_data_structure=self._get_rendering_data_structure,
+                ).sudo().run()
+            except MissingError:
+                _logger.warning("Unexpected domain [%s] in action server, interpreted as False", expression(domain, self).result)
```

Comment:
_logger.warning("The provided domain [%s] in ir.actions.server generated a MissingError in '%s'", expression(domain, self).result, self._name)

Reply:
Done :+1: